.github/workflows/e2e-tests.yml

```diff
@@ -12,6 +12,8 @@
 
 name: E2E Tests
 on:
+  schedule:
+    - cron: '0 16 * * *'
```

Comment:
Integration tests use `0 4 * * *` and use the same account. I suggest to split load and start the e2e tests 12 hours later.

Reply:
We should avoid running during US daytime preferably. Maybe start a couple hours after integration tests start?